test/e2e/e2e.go

```diff
@@ -1871,3 +1872,223 @@ func getNodePodCIDR(nodeName string) (string, error) {
+			errStr        string
+		)
+		checkDbFilesExistence := func(db_pod_num int) {
+			out, _ := framework.RunKubectl(ovnNs, "exec", db_pod_name, containerFlag, "--", "ls", dirDB)
+			out = strings.TrimSuffix(strings.ReplaceAll(out, "\n", ","), ",")
+			framework.Logf("current db files:%s", out)
+			ginkgo.By("current db files:" + out)
+			errStr = ""
+			if !strings.Contains(out, northDBFileName) {
+				errStr += " " + northDBFileName
+			}
+			if !strings.Contains(out, southDBFileName) {
+				errStr += " " + southDBFileName
+			}
+			if errStr != "" {
+				errStr = "missing db:" + errStr
+				framework.Failf(errStr)
+			}
+		}
+		checkDbFilesExistence(db_pod_num)
+		//delete db file(s)
+		for _, DBFileNameToDelete := range DBFileNamesToDelete {
+			framework.Logf("deleting db file %s", DBFileNameToDelete)
+			framework.RunKubectl(ovnNs, "exec", db_pod_name, containerFlag, "--", "rm", dirDB+DBFileNameToDelete)
+			//check db files- make sure its deleted
+			out, _ := framework.RunKubectl(ovnNs, "exec", db_pod_name, containerFlag, "--", "ls", dirDB)
+			out = strings.TrimSuffix(strings.ReplaceAll(out, "\n", ","), ",")
+			if out == "" {
+				framework.Logf("all db files deleted")
+			} else {
+				framework.Logf("cur db files:%s", out)
+			}
+			if strings.Contains(out, DBFileNameToDelete) {
+				errStr = "db not deleted:" + DBFileNameToDelete
+				framework.Failf(errStr)
+			}
+		}
+
+		//delete the db-pod in order to emulate the cluster/pod restart
+		framework.Logf("deleting db pod:%s", db_pod_name)
+		deletePod(f, ovnNs, db_pod_name)
+
+		//wait (let the new db pod to be ready)
+		framework.Logf("wait %v", waitingPeriod)
+		time.Sleep(waitingPeriod)
+
+		//check db files existence
+		checkDbFilesExistence(db_pod_num)
+	}
+	singlePodConnectivityTest := func(f *framework.Framework, podName string) {
+		framework.Logf("Running container which tries to connect to 8.8.8.8 in a loop")
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+
+		go checkContinuousConnectivity(f, "", podName, "8.8.8.8", 53, 30, podChan, errChan)
+		testPod := <-podChan
+
+		framework.Logf("Test pod running on %q", testPod.Spec.NodeName)
+		framework.ExpectNoError(<-errChan)
+	}
+	twoPodsConnectivityTest := func(f *framework.Framework,
+		node1Name string, node2Name string,
+		syncChan chan string, errChan chan error) {
+		const (
+			pod1Name                  string        = "connectivity-test-pod1"
+			pod2Name                  string        = "connectivity-test-pod2"
+			port                      string        = "8080"
+			timeIntervalBetweenChecks time.Duration = 2 * time.Second
+		)
+
+		var (
+			command = []string{"/agnhost", "netexec", fmt.Sprintf("--http-port=" + port)}
+		)
+		createGenericPod(f, pod1Name, node1Name, f.Namespace.Name, command)
+		createGenericPod(f, pod2Name, node2Name, f.Namespace.Name, command)
+
+		// wait for pods setup to return a valid address
+		wait.PollImmediate(retryInterval, retryTimeout, func() (bool, error) {
```

Comment:
I realize this is copied from other tests.. but dang, a pod that is Running should... be up.

(This can stay, just commentary).

Reply:
You're right.
I thought this test is necessary but it's redundant.
anyway, I removed this part.
thanks!